README.rst

```diff
@@ -108,13 +108,13 @@ Then, execute::
 
 From source on Windows
 ~~~~~~~~~~~~~~~~~~~~~~
+ 
+Install the dependencies with conda (Anaconda or Miniconda)::
 
-Install the dependencies with conda::
-
-   conda install numpy cython future six setuptools mkl
+   conda.exe install -c conda-forge numpy cython future six setuptools mkl
 
 Additionally, make sure you have a C compiler setup to compile Python C
-extensions, e.g.  Visual C++.
+extensions, e.g.  Visual C++. Build tools for VS2019 https://visualstudio.microsoft.com/downloads/#build-tools-for-visual-studio-2019 have been tested to work for this (see https://github.com/matthias-k/cyipopt/issues/52).
 
```

Comment:
Shouldn't this be stated wrt to a specific Python version? We are still supporting 2.7 and 3.6+.